src/common/Logo.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+
+const Logo = () => {
+  return (
+    <svg
```

Comment:
Some of the images looked blurry, some of them needed to be styled. In this case I believe I had responsive issues using Box with hardcoded styles and svg path, instead of Logo/svg